content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -653,73 +728,80 @@ This is commonly used by add-on API servers for unified authentication and autho
-* system:controller:root-ca-cert-publisher
-* system:controller:route-controller
-* system:controller:service-account-controller
-* system:controller:service-controller
-* system:controller:statefulset-controller
-* system:controller:ttl-controller
-
-## Privilege Escalation Prevention and Bootstrapping
+* `system:controller:attachdetach-controller`
+* `system:controller:certificate-controller`
+* `system:controller:clusterrole-aggregation-controller`
+* `system:controller:cronjob-controller`
+* `system:controller:daemon-set-controller`
+* `system:controller:deployment-controller`
+* `system:controller:disruption-controller`
+* `system:controller:endpoint-controller`
+* `system:controller:expand-controller`
+* `system:controller:generic-garbage-collector`
+* `system:controller:horizontal-pod-autoscaler`
+* `system:controller:job-controller`
+* `system:controller:namespace-controller`
+* `system:controller:node-controller`
+* `system:controller:persistent-volume-binder`
+* `system:controller:pod-garbage-collector`
+* `system:controller:pv-protection-controller`
+* `system:controller:pvc-protection-controller`
+* `system:controller:replicaset-controller`
+* `system:controller:replication-controller`
+* `system:controller:resourcequota-controller`
+* `system:controller:root-ca-cert-publisher`
+* `system:controller:route-controller`
+* `system:controller:service-account-controller`
+* `system:controller:service-controller`
+* `system:controller:statefulset-controller`
+* `system:controller:ttl-controller`
+
+## Privilege escalation prevention and bootstrapping
 
 The RBAC API prevents users from escalating privileges by editing roles or role bindings.
 Because this is enforced at the API level, it applies even when the RBAC authorizer is not in use.
 
-A user can only create/update a role if at least one of the following things is true:
+### Restrictions on role creation or update
 
-1. They already have all the permissions contained in the role, at the same scope as the object being modified
-(cluster-wide for a `ClusterRole`, within the same namespace or cluster-wide for a `Role`)
-2. They are given explicit permission to perform the `escalate` verb on the `roles` or `clusterroles` resource in the `rbac.authorization.k8s.io` API group (Kubernetes 1.12 and newer)
+You can only create/update a role if at least one of the following things is true:
 
-For example, if "user-1" does not have the ability to list secrets cluster-wide, they cannot create a `ClusterRole`
+1. You already have all the permissions contained in the role, at the same scope as the object being modified
+(cluster-wide for a ClusterRole, within the same namespace or cluster-wide for a Role).
+2. You are granted explicit permission to perform the `escalate` verb on the `roles` or `clusterroles` resource in the `rbac.authorization.k8s.io` API group.
+
+For example, if `user-1` does not have the ability to list Secrets cluster-wide, they cannot create a ClusterRole
```

Comment:
The style guide would pick Secrets here, and I think that makes sense. It also helps localization teams understand what text to preserve as Kubernetes resource names. Is it OK to keep “Secrets”?

Reply:
Same comment about being consistent with the URL and the way the resource is referred to in the role definition. We see people make mistakes a lot when defining roles, we just tried to refer to resources that permissions were being granted to in the same way throughout this doc to reinforce the form used.

Reply:
It's not the end of the world to change it, but there was a reason for it being the way it was, in case that helps justify a local departure from a global style

Reply:
I've decided to write (eg) both ConfigMap and `"configmaps"` and add text that I hope helps readers to understand which goes where.